acceptance/localcluster/docker.go

```diff
@@ -146,6 +146,12 @@ func maybePanic(err error) {
 // Remove removes the container from docker. It is an error to remove a running
 // container.
 func (c *Container) Remove() error {
+	if os.Getenv("CIRCLE_ARTIFACTS") != "" {
+		// HACK: Removal of docker containers fails on circleci with the error:
```

Comment:
this is because of CircleCI's setup. I had asked their support and they said something about it being due to docker running pseudo-root.